lib/rubycritic/platforms/darwin.rb

```diff
@@ -0,0 +1,27 @@
+require "rubycritic/platforms/base"
+
+module Rubycritic
+  module Platforms
+    class Darwin < Rubycritic::Platforms::Base
+      def self.can_open?
+        platform =~ /darwin/
+      end
+
+      def application
+        if browser_name =~ /chrome/i
```

Comment:
We are mixing code and data here - how about just using a hash here and get rid of those ugly "if"s?